test/langtools/tools/javac/patterns/Unnamed.java

```diff
@@ -0,0 +1,255 @@
+import java.util.Objects;
+
+/**
+ * @test /nodynamiccopyright/
+ * @bug 8304246
+ * @summary Compiler Implementation for Unnamed patterns and variables
+ * @enablePreview
+ * @compile --enable-preview -source ${jdk.version} Unnamed.java
```

Comment:
+1 on keeping the `@compile` tag, but `--enable-preview -source ${jdk.version}` shouldn't be needed when `@enablePreview` is present.